src/acp/acp.test.ts

```diff
@@ -32,21 +32,22 @@ jest.mock("../fetcher.ts", () => ({
 }));
 
 import { Response } from "cross-fetch";
-import {
-  getFileWithAcp,
-  getResourceInfoWithAcp,
-  getSolidDatasetWithAcp,
-  WithAccessibleAcr,
-} from "./acp";
-import { acp, rdf } from "../constants";
 import * as SolidDatasetModule from "../resource/solidDataset";
 import * as FileModule from "../resource/nonRdfData";
 import * as ResourceModule from "../resource/resource";
-import { mockSolidDatasetFrom } from "../resource/mock";
-import { File, UrlString, WithServerResourceInfo } from "../interfaces";
-import { AccessControlResource } from "./control";
+import {
+  getFileWithAcr,
+  getReferencedPolicyUrls,
```

Comment:
Minor point - but for consistency, should this be named `getReferencedPolicyUrlAll()` (i.e. is this literally the only function name with a plural)...?

Reply:
Good point: f585e8f2293a5e1184f5436fe2148f4ae06a54d8